src/types.ts

```diff
@@ -3,6 +3,7 @@ export interface FarosClientConfig {
   readonly apiKey: string;
   readonly useGraphQLV2?: boolean;
   readonly phantoms?: Phantom;
+  readonly visibility?: string;
```

Comment:
What values can this have? If it's a small static list, then can you type it as a string literal union. Otherwise string is fine.

Reply:
there is no constraint on the values.  it is a user-defined value that just needs to match for write and read APIs